src/client/components/modals/CubeOverviewModal.tsx

```diff
@@ -75,11 +75,11 @@ const CubeOverviewModal: React.FC<CubeOverviewModalProps> = ({ isOpen, setOpen,
       }),
     });
 
+    const body = await response.json();
     if (response.ok) {
       //Reload page to ensure state is updated
-      window.location.reload();
+      window.location.replace(body.redirect);
```

Comment:
Are there any security implications of this? I am returning a root relative path